core/sup_tree_core/gear_config_poller.ex

```diff
@@ -29,10 +33,21 @@ defmodule AntikytheraCore.GearConfigPoller do
     {:ok, %{last_checked_at: 0}, @interval}
   end
 
+  @impl true
+  def handle_cast(:reload, state) do
+    new_state = load_gear_config(state)
+    {:noreply, new_state, @interval}
```

Comment:
"Making a 3-tuple" is also duplicated for the 2 handler callbacks.
Let's add this part in the common private function (and rename it).